providers/oauth/oauth.httpobserver.js

```diff
@@ -35,14 +35,19 @@ FirefoxTabsAuth.prototype.initiateOAuth = function (redirectURIs, continuation)
   return false;
 };
 
-FirefoxTabsAuth.prototype.launchAuthFlow = function (authUrl, stateObj, continuation) {
+FirefoxTabsAuth.prototype.launchAuthFlow = function (authUrl, stateObj, interactive, continuation) {
   "use strict";
   var gBrowser = wm.getMostRecentWindow("navigator:browser").gBrowser,
     listener,
     complete,
+    fail,
     tab,
     previousTab = gBrowser.selectedTab;
 
+  if (interactive === undefined) {
```

Comment:
I think `typeof interactive === "undefined"` is probably a bit better/more robust.